### DISCUSSION TITLE
SMS 2FA not working

### DISCUSSION THREAD

**Discussion by bh-public** [community/community#44020] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

2FA SMS is not sending a code to my phone, this happens on both my github accounts.  This did work for me about a month ago, so I am wondering if there is a problem?

**Comment by genidma** [comment]
Note: This may not be the advise that you are looking for. Also, I do not work for Github. That being said and other notes/warnings below directly from Github's knowledge base.
- You can try switching from SMS to TOTP mobile app. But see note about 'recovery codes' first -> [link here](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/changing-two-factor-authentication-delivery-methods-for-your-mobile-device) 
- You can also try disabling 2FA. Logging out of the system. Logging back into the system and enabling 2FA again. 
- But be mindful of the warning via Github (officially) --> [link here](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/disabling-two-factor-authentication-for-your-personal-account) prior to doing so.
- Also, you mentioned SMS. Maybe you accidentally added the source to spam? So maybe contact your carrier.

**Comment by bh-public** [comment]
I called AT&T They ran me thru a few steps, reboot and now SMS is working
again.  It probably has to do with all the rain and wind and cell towers
going out with the crazy weather in Nor Cal.

**Reply by genidma** [reply]
Reminds me of the [ITcrowd :)](https://www.youtube.com/watch?v=EKtPWGvmAXw)
Glad that it's working now! Cheers!

**Reply by genidma** [reply]
Also, if you could please resolve this issue. Thank you.
